tooling/src/main/java/org/opencds/cqf/tooling/operations/stripcontent/ContentStripperR4.java

```diff
@@ -0,0 +1,25 @@
+package org.opencds.cqf.tooling.operations.stripcontent;
+
+import org.hl7.fhir.convertors.factory.VersionConvertorFactory_40_50;
+import org.hl7.fhir.r4.model.Resource;
+
+import ca.uhn.fhir.context.FhirContext;
+
+class ContentStripperR4 extends BaseContentStripper<Resource> {
+    @Override
+    protected FhirContext context() {
+        return FhirContext.forR4Cached();
+    }
+
+    // NOTE: These two methods appear to be identical, but they are not. It's the
+    // types of the input and output parameters that are different.
+    @Override
+    protected org.hl7.fhir.r5.model.Resource convertToR5(Resource resource) {
```

Comment:
We have utility functions for resource and type conversions that could be reused. See org.opencds.cqf.tooling.utilities.converters.ResourceAndTypeConverter

Reply:
Updated to use the utility functions.